scipy/_lib/tests/test__util.py

```diff
@@ -445,7 +445,7 @@ def f2(*args):
             ref2 = ref2.reshape(result_shape)
             ref3 = ref3.reshape(result_shape)
 
-        xp_assert_close(res1, ref1, rtol=2e-16, check_0d=False)
-        xp_assert_equal(res2, ref2, check_0d=False)
+        xp_assert_close(res1, ref1, rtol=2e-16)
+        xp_assert_equal(res2, ref2)
         if not is_array_api_strict(xp):
-            xp_assert_equal(res3, ref3, check_0d=False)
+            xp_assert_equal(res3, ref3)
```

Comment:
I'm not sure why these had `check_0d=False`? Tests pass for me locally without them.